README.rst

```diff
@@ -42,28 +42,51 @@ If you wan to run unit tests:
 Specifications
 --------------
 
-Point Cloud
-~~~~~~~~~~~
-
-Point Cloud Tile Format:
-https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/PointCloud
-
-.. image:: https://raw.githubusercontent.com/Oslandia/py3dtiles/master/docs/pc_layout.png
-    :width: 700px
-    :align: center
+Generic Tile
+~~~~~~~~~~~~
 
 The py3dtiles module provides some classes to fit into the
 specification:
 
 - *Tile* with a header *TileHeader* and a body *TileBody*
-- *TileHeader* represents the first 28 bytes (magic value, version, ...)
-- *TileBody* contains the feature table *FeatureTable* and the batch table (not supported for now)
-- *FeatureTable* with a header *FeatureTableHeader* and a *FeatureTableBody*
-- *FeatureTableBody* which contains features of type *Feature*
+- *TileHeader* represents the metadata of the tile (magic value, version, ...)
+- *TileBody* contains varying semantic and geometric data depending on the the tile's type
 
-Moreover, a utility class *TileReader* is available to read a *.pnts*
+Moreover, a utility class *TileReader* is available to read a tile
 file as well as a simple command line tool to retrieve basic information
-about a Point Cloud file **py3dtiles\_info**.
+about the tile **py3dtiles\_info**.
```

Comment:
How about adding a column after "tile" here? Also, I'd use "about a tile" rather than "about the tile".